app/views/posts/show.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Here is a list of posts give by a user and post id</h1>
+<div class="post-container">
+    <div class="post-header">
+            <h2><%= @post.title%> By <%= @post.author.name%></h2>
+            <ul>
+                <li>Comments: <%= @post.comment_counter%> </li>
+                <li>Likes: <%@post.like_counter%> </li>
+            </ul>
+    </div>
+
+    <h4 class="comment-title">Comments</h4>
+    <div class="post-body">
+        <%=@post.text%>
+    </div>
+
+    <div class="comment-container">
+        <% @post.recent_comments.each do |comment| %>
+            <%= comment.user.name%> : <%= comment.text%>
+        <% end %>
```

Comment:
Hello @codepantha sir. Can we meet for 10 minutes really quick sir . I have some questions to ask about updating my model and I would like to know whch changes do I need to make so that I can create new comments . I would really appreciate it sir . Thanks again for the constructive feedback 😁